lib/rules/shorthand-property-no-redundant-values/index.mjs

```diff
@@ -138,6 +138,16 @@ const rule = (primary, _secondaryOptions, context) => {
 					break;
 				}
 
+				// `var()` should be ignored. E.g.
+				//
+				//   --padding: 20px 5px;
+				//   padding: 0 var(--padding) 0;
+				//
+				if (isValueFunction(valueNode) && valueNode.value.toLowerCase() === 'var') {
+					shouldIgnore = true;
+					break;
+				}
```

Comment:
suggestion
refactor

What about a `return` inside both this `if` and the preceding one?
i.e. `shouldIgnore` flag seems unnecessary

Reply:
Nice catch. Fixed via 101003d36 👍🏼